### PR TITLE
Ks/ktlint integration

### PR DESCRIPTION
**Pull request by KevinSchildhorn**
Adding Ktlint to Kermit and it's sample.

### CODE REVIEW

File: KermitSample/.idea/compiler.xml [hunk 1]
```diff
@@ -4,6 +4,11 @@
     <annotationProcessing>
```
Comment (russhwolf) [inline KermitSample/.idea/compiler.xml:4]: I think this file should be ignored

File: KermitSample/.idea/jarRepositories.xml [hunk 2]
```diff
@@ -46,5 +46,10 @@
       <option name="name" value="MavenLocal" />
```
Comment (russhwolf) [inline KermitSample/.idea/jarRepositories.xml:46]: I think this file should be ignored

File: KermitSample/.idea/compiler.xml (hunk 1 above)
Comment (samhill303) [reply KermitSample/.idea/compiler.xml]: Updated

**Review by mrf7 — APPROVED** [review]

**Comment by mrf7** [comment]
Closing to pull ktlint stuff away from CI/CD and merge in separate PRs

**Pull request closed without merging**